packages/webcert/src/components/header/WebcertHeaderUnit.tsx

```diff
@@ -4,20 +4,45 @@ import { getUser } from '../../store/user/userSelectors'
 import { shallowEqual, useSelector } from 'react-redux'
 import { User } from '@frontend/common/src'
 import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons'
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  font-size: 13px;
+`
 
 const Italic = styled.span`
   font-style: italic;
-  font-size: 13px;
+`
+
+const InactiveUnit = styled.span`
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+  cursor: help;
 `
 
 const WebcertHeaderUnit: React.FC = () => {
   const user = useSelector(getUser, shallowEqual)
 
   const toString = (user: User): React.ReactNode => {
     return (
-      <p>
-        {user.loggedInCareProvider.unitName} <br /> <Italic>{user.loggedInUnit.unitName}</Italic>
-      </p>
+      <Wrapper>
+        <span>
+          {user.loggedInCareProvider.unitName} <br />
+          <Italic>{user.loggedInUnit.unitName}</Italic>
+        </span>
+        {user.loggedInUnit.isInactive ? (
+          <InactiveUnit
+            className="iu-ml-400"
+            data-tip="Enheten är markerad som inaktiv i journalsystemet, vilket innebär att viss funktionalitet ej är tillgänglig.">
+            <FontAwesomeIcon className="iu-fs-300 iu-mr-200" icon={faExclamationCircle} size="1x" />
+            <span>Inaktiv enhet</span>
```

Comment:
Shouldn't the text be: OBS! Enheten är markerad som inaktiv i journalsystemet.

Reply:
I've synced with Jan, Anna and Kristina and we decided to implement it like this. I will update the Jira with this info and they will update the requirements